src/RomModule.h

```diff
@@ -50,9 +56,28 @@ class RomModule: public PeripheralDevice, public ChipPIO8255
 
 		void ReadFromRom();
 
-	private:
+	protected:
 		BYTE *RomPack;
 };
+//---------------------------------------------------------------------------
+class MegaModule: public RomModule

```

Comment:
Please, separate `MegaModule` to own source+header to keep it clean.

Reply:
Ok, I will do that. (however I'm not sure how to do it here on github ... ;-)

Reply:
You could do it on your machine and just commit+push, GitHub will know about it immediately. 😉